config/common/essentials.yaml

```diff
@@ -0,0 +1,13 @@
+template:
+  type: http
+  url: "{{stack_group_config.aws_infra_templates_root_url}}/v0.3.8/templates/essentials.yaml"
+stack_name: essentials
+
+parameters:
+  VpcPeeringRequesterAwsAccountId: '745159704268'
```

Comment:
Not sure if this is relevant anymore with the new TGW setup, but it's a required parameter, so I don't have a choice.

Reply:
newer accounts (created with org-formation) should need essentials.yaml.  it creates a bunch of resources that are not really needed anymore.  If maintenance depends on essential resources then IT needs to fix that.

Reply:
looking at the [maintenance template](https://github.com/Sage-Bionetworks/aws-infra/blob/master/templates/maintenance.yaml), I don't see any dependencies on resources from essentials.yaml.  Maybe all you need to do is drop essentials.yaml from this PR.